src/tasks/buildArtifact/BuildArtifactTaskProvider.ts

```diff
@@ -206,7 +206,7 @@ class ExportJarTaskTerminal implements Pseudoterminal {
 
     public exit(message?: string) {
         if (message) {
-            this.writeEmitter.fire(message);
+            this.writeEmitter.fire(message + "\r\n");
```

Comment:
If `message` is undefined, will the output look good?

Reply:
if message is undefined, we will not invoke writeEmitter so nothing will happen in the outout panel.